libclc/amdgcn-amdhsa/libspirv/math/sincos.cl

```diff
@@ -0,0 +1,60 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+  #include <clcmacro.h>
+  #include <spirv/spirv.h>
```

Comment:
Interesting... clang-format doesn't complain here - all code except the first comment block is shifted by 2 spaces.